package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "genesys-react-components",
-	"version": "1.1.0",
+	"version": "1.2.0",
```

Comment:
I didn't update the app folder since this version isn't published yet, figured I would open a second PR for that once this version gets published